examples/selenium/selenium-hub-rc.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: selenium-hub 
+  labels:
+    name: selenium-hub
```

Comment:
please use 'app' instead of 'name' as it is redundant (and confusing) with `metadata.name`